docs/contribute/processes/development.md

```diff
@@ -2,13 +2,41 @@
 title: Development Process
 ---
 
-Our development workflow bases on the standard [GitHub workflow](https://guides.github.com/introduction/flow/) and consists of the following steps:
+Our development workflow bases on the standard [GitHub workflow](https://guides.github.com/introduction/flow/) with the slight variation that developers are working on forks of the main repository.
```

Comment:
I kept the link to the default github workflow as it explains the branching workflow. But that is something that we still have to decide on: Should external developers create branches that are just shipped to out repo by PR and we merge the branches into the master or should external developers create PRs onto master?

If the second option should be used (PR onto master), just linking to the forking documentation is easier. Then we would only have to mention the review process and CI stuff. If the first option should be used (PR onto new branch, core developer "merges" branch into master), we still have to explain this workflow.

The issue with the second workflow is that we require PRs to be up to date with the current master, meaning the timing of getting the developer to rebase and a core developer to merge could get tricky if there is a lot of activity on the master.

On the other hand, being up to date with the master is something that we can't enforce with the first workflow. Furthermore, potential merge conflicts then have to be solved by the core developer merging the branch into master instead of the actual PR developer.

Currently, I am leaning towards second workflow (external PR onto master). What are your opinions?